api/src/services/lottery.service.ts

```diff
@@ -956,6 +974,64 @@ export class LotteryService {
     return formattedActivityLogs;
   }
 
+  /**
+    runs the job to auto expire lotteries that are passed their due date
+    will call the the cache purge to purge all listings as long as updates had to be made
+  */
+  async autoPublishResults(): Promise<SuccessDTO> {
+    this.logger.warn('autoPublishLotteryResults job running');
+    await this.listingService.markCronJobAsStarted(
+      LOTTERY_PUBLISH_CRON_JOB_NAME,
+    );
+    const tomorrow = dayjs(new Date()).add(1, 'days').toDate();
+    const releasedListings = await this.prisma.listings.findMany({
+      select: {
+        id: true,
+        name: true,
+        jurisdictions: true,
+      },
+      where: {
+        status: ListingsStatusEnum.closed,
+        reviewOrderType: ReviewOrderTypeEnum.lottery,
+        lotteryOptIn: true,
+        lotteryStatus: LotteryStatusEnum.releasedToPartners,
```

Comment:
Would be interested to check w product on whether or not this should also happen if it hasn't been released

Reply:
The AC in the ticket specified only when it has been released